CMakeLists.txt

```diff
@@ -23,11 +23,58 @@ project(pflib
 	DESCRIPTION "Pretty fine HGCROC configuration library."
 	LANGUAGES CXX)
 
+
 # YAML->register value "compiler"
 #   optional, build it if we can find the yaml parser we use
 find_package(yaml-cpp REQUIRED)
 message(STATUS "Found yaml-cpp ${yaml-cpp_VERSION}")
 
+set(
+  ROC
+  "sipm_rocv3b"
```

Comment:
What should the default be? I think we have SiPM ROCv3b for the Hcal test stands, but I'd like to confirm that.

Reply:
Hmmm... Yeah, this is probably true, though I was wondering if we could just build them all and pick the right one at runtime?

Reply:
Yea, I can do that